libsql-server/tests/cluster/mod.rs

```diff
@@ -104,8 +109,6 @@ fn proxy_write() {
             Value::Integer(1)
         ));
 
-        snapshot_metrics().assert_gauge("libsql_server_current_frame_no", 2.0);
```

Comment:
Why is this not good any more?

Reply:
These can be added back they were removed 1) because we are now sure metrics emit 2) there were race conditions with how the previous implementation used namespaces that were causing different outcomes on CI and locally. I will add these back since I reworked the implementation to not use a namespace.